pkg/comp-functions/functions/vshnnextcloud/files/backup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+# We need to run the occ command as the www-data user, but there's no sudo by default
+apt update 1>&2 && apt install sudo -y 1>&2
+
+function disableMaintenance {
+  >&2 echo "Disabling maintenance"
+  sudo -u www-data /var/www/html/occ maintenance:mode --off 1>&2
+}
+
+trap disableMaintenance EXIT
```

Comment:
I just learned what this `trap` is about

Reply:
> A day without learning is a day wasted. There is so much to learn and so little time to learn it.

:D